### PR TITLE
Reduce the replicaCount

### PR DESCRIPTION
**Pull request by seansund**

### CODE REVIEW

File: inventory-svc/values.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 global: {}
 
 inventory-svc:
-  replicaCount: 3
+  replicaCount: 1
```
Comment (csantanapr) [inline inventory-svc/values.yaml:4]: Are you sure is for staging?

**Review by csantanapr — CHANGES_REQUESTED** [review]

File: inventory-svc/values.yaml (hunk 1 above)
Comment (seansund) [reply inventory-svc/values.yaml:4]: Yes

Comment (csantanapr) [reply inventory-svc/values.yaml:4]: I think we need to ask Roland abou this

Comment (csantanapr) [reply inventory-svc/values.yaml:4]: Fine Fine I will merge it

**Review by csantanapr — APPROVED** [review]
LGTM

**Pull request merged**